packages/s2-core/src/common/interface/s2Options.ts

```diff
@@ -331,6 +331,10 @@ export interface S2PivotSheetOptions {
    */
   hierarchyType?: HierarchyType;
 
+  /**
+   * 是否配置线性样式，默认不配置
+   */
+  treeLineShow?: boolean;
```

Comment:
```suggestion
  showTreeLine?: boolean;
```

另外连接线属于行头的能力, 建议放在 `s2Options.style.rowCell.showTreeLine`